EssentialFeed/EssentialFeed/Image Comments/API/ImageCommentsMapper.swift

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+public final class ImageCommentsMapper {
+	private struct Root: Decodable {
+		private let items: [Item]
+
+		private struct Item: Decodable {
+			let id: UUID
+			let message: String
+			let created_at: Date
+			let author: Author
+		}
+
+		private struct Author: Decodable {
+			let username: String
+		}
+
+		var comments: [ImageComment] {
+			return items.map { ImageComment(id: $0.id,
+			                                message: $0.message,
+			                                createdAt: $0.created_at,
+			                                username: $0.author.username) }
+		}
+	}
+
+	static public func map(_ data: Data, from response: HTTPURLResponse) throws -> [ImageComment] {
```

Comment:
The convention in this codebase is to define the access control first, before `static`:

```suggestion
	public static func map(_ data: Data, from response: HTTPURLResponse) throws -> [ImageComment] {
```